### PR TITLE
Add support for django-hijack

### LINKED ISSUE #9: Make two-factor auth work with django-hijack
Hopefully hijack sends a signal (or sets user.backend) so that we could perhaps store/use `user.backend` and add the hijack "backend" to the bypass allow list.

Perhaps it would be good to enforce the hijacking user to have 2FA set up themselves and re-authenticate?

### PR DESCRIPTION
**Pull request by sergei-maertens**
Closes #9

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -48,3 +57,41 @@ def recovery_codes(user) -> list[str]:
     ]
     StaticToken.objects.bulk_create(tokens)
     return [token.token for token in tokens]
+
+
+@pytest.fixture
+def mfa_admin_user(admin_user):
```
Comment (Viicos) [inline tests/conftest.py:63]: Is `admin_user` a pytest-django specific fixture?

**Review by Viicos — APPROVED** [review]
Is this something that also should be taken into account in the upstream django two factor auth library? Or is it really specific to MM?

File: tests/conftest.py (hunk 1 above)
Comment (sergei-maertens) [reply tests/conftest.py:63]: yep!

**Comment by sergei-maertens** [comment]
> Is this something that also should be taken into account in the upstream django two factor auth library? Or is it really specific to MM?

Really specific to our particular combination of django-2fa and django-hijack.

Of course, users of django-two-factor-auth can always use this library as a source of inspiration :D

**Pull request merged**